.github/workflows/ci-image-diff-sync.yml

```diff
@@ -47,13 +47,13 @@ jobs:
       XSS_PROTECTION: True
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - uses: actions/setup-python@v2
       with:
         python-version: 3.9.9
-    - uses: actions/setup-node@v2
+    - uses: actions/setup-node@v3
       with:
-        node-version: 18
+        node-version: 16
```

Comment:
Not sure if we wanna stick to `16` for consistency with `actions/checkout@v3`'s node version (16) or 18

https://github.com/actions/setup-node#usage

Reply:
I think we should stick with '18' to make sure we use the same version locally, in CI and on staging and prod.